Adyen/UI/List/ListItem.swift

```diff
@@ -10,6 +10,8 @@ import Foundation
 @_spi(AdyenInternal)
 public class ListItem: FormItem {
```

Comment:
Are there any `FormItem` elements left that do not have this behavior?

Reply:
every `FormItem` now has a `public var isHidden: AdyenObservable<Bool>` property